src/h2load.cc

```diff
@@ -1199,6 +1319,12 @@ Worker::~Worker() {
   ev_loop_destroy(loop);
 }
 
+void Worker::stop_all_clients() {
+  for(auto client: clients) {
+    client->~Client();
```

Comment:
So, there might be some confusions regarding my last comment. The ownership is modeled like the following, I think: The worker threads virtually owns the clients, but not in reality. The clients are created in `run` function of the `Worker` class. However, there was no member variables in the `Worker`class, for the clients it would create.
I added a member variable. I don't know if the community will approve my style, but with the first look at RAII, I think this new approach is more in line with that philosophy.